cylc/flow/scripts/lint.py

```diff
@@ -134,7 +135,7 @@
         'url': STYLE_GUIDE + 'indentation',
         'index': 5
     },
-    re.compile(r'\s$'): {
+    re.compile(r'[ \t]$'): {
```

Comment:
All other whitespace (e.g: ` `) are ok?

Reply:
Before, the code stripped the newline before running it through the check, then put the newline back if `modify == True` afterwards.

Now the newline character is left alone. Because `\s` includes `\n` this caused every line to be marked as containing trailing whitespace.

Reply:
This will do then - If anyone wishes to go searching for obscure whitespace characters that can be their problem.